package.json

```diff
@@ -5,9 +5,9 @@
   "license": "GPL-3.0",
   "scripts": {
     "webpack": "NODE_ENV=production webpack -p --bail",
-    "start": "webpack-dev-server --compress",
-    "start:noc": "webpack-dev-server",
-    "startstage": "API_HOST=http://hchung.westus.cloudapp.azure.com:5000 npm start",
+    "start": "webpack-dashboard -- webpack-dev-server --inline --compress",
```

Comment:
Webpack dashboard is just a cool console visualization for webpack. That being said, I am not sure we should make it the default npm start because it has some issues scrolling properly in iterm.